src/components/dialog/Dialog.jsx

```diff
@@ -85,7 +84,7 @@ function BaseDialog({
    * The name of transition with the longest duration, because
    * a component can have an animation of many properties.
    */
-  const lastTransitionName = exiting || reduceMotion ? 'opacity' : 'transform';
+  const lastTransitionName = exiting ? 'opacity' : 'transform';
```

Comment:
In case of `motionPreset === 'none'` there is not transition, so I guess we need to handle this here?

Reply:
if there is no transition we need to handle it here 

``` 
if (!supportsTransitions() || motionPreset === 'none') {
      fireTransitionEndCallbacks();
    }
  }, [open, fireTransitionEndCallbacks, motionPreset]);```

Reply:
And for now it is fine but when we introduce other motion presets the last transition may changed

Reply:
we will end up with some mapping I guess like 
```
Record<MotionPreset, TransitionName>
```